lib/src/analyzers/lint_analyzer/rules/rules_list/prefer_correct_edge_insets_constructor/prefer_correct_edge_insets_constructor_rule.dart

```diff
@@ -12,6 +12,10 @@ import '../../../models/replacement.dart';
 import '../../../models/severity.dart';
 import '../../models/flutter_rule.dart';
 import '../../rule_utils.dart';
+import 'models/edge_insets_data.dart';
+import 'models/edge_insets_param.dart';
+
+part 'validator.dart';
 
```

Comment:
Could you group part directives together (remove blank line)?

Reply:
Done